src/geo_templates/default/main_page/alternate_front_pages/showcase.tpl

```diff
@@ -205,18 +205,16 @@
 						</div>
 					</div>
 				<!-- END FEATURED CAROUSEL -->
-						
+
 				<!-- BANNER BEGIN -->
 				{* Example place for 300x100 image banners *}
-				<div class="content_box cntr banner">	         
-					<a href="http://www.kqzyfj.com/click-7066924-11191670" target="_blank"><img src="{external file='images/banners/1and1_300x100.jpg'}" alt="1and1 Hosting" title="1and1 Hosting" /></a>
-					<a href="http://www.hostmonster.com/track/geodesic/resources" target="_blank"><img src="{external file='images/banners/hostmonster_300x100.jpg'}" alt="HostMonster Hosting" title="HostMonster Hosting" /></a>
-					<a href="http://lunarpages.com/id/geodesic/goto/basic" target="_blank"><img src="{external file='images/banners/lunarpages_300x100.jpg'}" alt="Lunarpages.com Hosting" title="Lunarpages.com Hosting" /></a>
+				<div class="content_box cntr banner">
+					<a href="https://geodesicsolutions.org/" target="_blank"><img src="{external file='images/banners/sample_300x100.jpg'}" alt="Sample Ad Banner" title="Sample Ad Banner" /></a>

```

Comment:
Be sure to delete the old images (after double checking that they are not used anywhere).

If someone has an old version of the templates, they should also have the old files as well, so that isn't a concern.

Reply:
I considered that but decided to leave them there. My experience tells me that cleaning up things often leads to breaking things because you ultimately miss something. They may also be needed for one of the alternate template sets. There are many banners in that folder that were not referenced in the default templates I changed. The original scope was to fix the default templates.

I'll remove the ones that were referenced in the default templates I touched.

Reply:
Heh I re-learned that earlier, I'm working on cleaning up the user edit in the admin and started to delete a function that was not called anywhere... Forgetting that `display_` and `update_` functions are called by the system based on the URL.  So those you would not find a call anywhere but they are still used sometimes.

Anyways, in this case, it is an image, so it should be safe.

Worst case, you miss somewhere, and we either notice it when testing, or it is somewhere no one notices and ends up not mattering much anyways 😆 .

Regarding deleting things in general:  it is better to remove something that is no longer used at the time it stops getting used.  For instance, it is kind of a common thing for a dev to leave some code commented out "just in case they need it" - well that ends up with cluttered code, you are left in a month or a year wondering what this commented out line is for.  And if you need it or not.  Vs. just deleting - if it turns out you need something bad enough, and can't just re-code it, that is what git history is for.

But I digress, it is good to be cautious but in this case, we should be fine removing images that are no longer used.  Especially images that are for "sample banners".